configure

```diff
@@ -323,10 +323,11 @@ valopt target "${CFG_HOST}" "GNUs ./configure syntax LLVM target triples"
 
 valopt localstatedir "/var/lib" "local state directory"
 valopt sysconfdir "/etc" "install system configuration files"
-valopt datadir "${CFG_PREFIX}/share" "install data"
-valopt infodir "${CFG_PREFIX}/share/info" "install additional info"
-valopt mandir "${CFG_PREFIX}/share/man" "install man pages in PATH"
-valopt libdir "${CFG_PREFIX}/lib" "install libraries"
+valopt datadir "share" "install data (relative to prefix)"
```

Comment:
So I'll explain my reasoning using the default value of `CFG_PREFIX` which is `/usr/local`. When using the `CFG_DATADIR` directory in the Makefile, it will cause everything to be installed into `/usr/local/share` but then when you go to install everything `CFG_PREFIX` will get passed to install.sh with `--prefix`. The result is anything installed to `CFG_DATADIR` is installed to `/usr/local/usr/local/share`